src/tests/Microsoft.PowerFx.Interpreter.Tests/RecalcEngineAsyncTests.cs

```diff
@@ -19,14 +19,16 @@ namespace Microsoft.PowerFx.Tests
     // Test async eval features. 
     public class RecalcEngineAsyncTests : PowerFxTest
     {
-        // Trivial async function that runs synchronously. 
+        // Intentionally trivial async function that runs synchronously. 
+#pragma warning disable CS1998 // Async method lacks 'await' operators and will run synchronously
```

Comment:
Can we remove this warning?
async keyword actually matters for how AggregateException is handled.